### DISCUSSION TITLE
Enable thumbs up/thumbs down

### DISCUSSION THREAD

**Discussion by ianlcassidy** [microsoft/sample-app-aoai-chatGPT#688] [Q&A] [answered]
I see that there is a frontend component in [frontend/src/components/Answer/Answer.tsx](https://github.com/microsoft/sample-app-aoai-chatGPT/blob/98d6d50247a1d6bc64a1fe477d085a5f40188426/frontend/src/components/Answer/Answer.tsx#L94) for thumbs up/thumbs down feedback, but I'm not seeing it show up in my app. Does anyone know how to "enable" this?

**Comment by sarah-widder** [comment, marked as answer]
Hi @ianlcassidy , if you have Chat History enabled, you just need to add an environment variable described here: https://github.com/microsoft/sample-app-aoai-chatGPT?tab=readme-ov-file#local-setup-enable-message-feedback

**Reply by ianlcassidy** [reply]
thanks @sarah-widder - i totally missed that! need to do a better job of RTFM next time 😭
